packages/components/src/GeometryPanel.component.svelte

```diff
@@ -1,32 +1,124 @@
 <svelte:options
   customElement={{
-    tag: 'geometry-panel'
+    tag: 'geometry-panel',
+    props: {
+      features: { type: 'Array' },
+			categories: {type: 'Array'},
+			attributes: {type: 'Array'},
+			targets: { type: 'Array' }
+    }
   }}
 />
 
 <script lang="ts">
 	import Select from './misc/Select.component.svelte';
 	import RenderingSettings from './geometry/Settings.component.svelte';
-  import { GUIEvent, SelectProps } from './types';
+	import CategoricalPanel from './misc/CategoricalPanel.component.svelte';

```

Comment:
Bad indentation?

Reply:
Fixed in subsequent [#147](https://github.com/TileDB-Inc/TileDB-Viz/pull/147)